tools/spelling/spellcheck.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2023-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from common import cspell, Problem, FIXME_FILE
+import json
+import re
+
+cspell_out = json.loads(cspell("--show-context", "--reporter", "@cspell/cspell-json-reporter"))
```

Comment:
Can you `if __name__ == "__main__":` this? And maybe provide some `--help` via argparse?

Reply:
Good point

Reply:
Done